src/api/web/api.js

```diff
@@ -45,11 +45,13 @@ module.exports = function(config, auth, storage) {
         packages,
         function(pkg, cb) {
           auth.allow_access(pkg.name, req.remote_user, function(err, allowed) {
-            if (err) {
-              cb(null, false);
-            } else {
-              cb(err, allowed);
-            }
+            async.setImmediate(function() {
```

Comment:
I wonder, why don't https://nodejs.org/api/timers.html#timers_setimmediate_callback_args instead?

Reply:
My point is. I'm trying to get rid of async library.

Reply:
My suggestion.
```
setImmediate(function() {
        if (err) {
             cb(null, false);
        } else {
            cb(err, allowed);
       }
});
```

Reply:
When I solving a bug I look at doc of async library and found a solution here (https://caolan.github.io/async/  Common Pitfalls -> Synchronous iteration functions). That's why I use function async.setImmediate. But I agree that solution with nodes setImmediate should also work.

Reply:
Thanks !

Reply:
I'm just waiting to close this PR to release a new version.